### DISCUSSION TITLE
Possibility to pause or put breakpoints in a simulation (to study the data graph) ?

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#982] [Q&A] [answered]
When I stop the simulation, the accumulated data in the data graph disappear. It would be nice to be able to pause the simulation  in order to look in detail at the data graph after a number of clock cycles into the simulation. Most likely it is already built in or can be realized with existing tools  ? Similar question for a breakpoint (set e.g. when one of the signals becomes high or low ?).

**Comment by hneemann** [comment, marked as answer]
It is also possible to enable a data graph in the settings which opens in its own dialog. This data graph should keep it's data.

**Reply by jongena** [reply]
Thank you for the quick answer. This is an interesting option, but I did not find it. ? Could you let me know how / where to switch this on ?

**Reply by hneemann** [reply]
In the moment I don't have a machine at hand, but if I remember correctly it's in the circuit specific settings.

**Reply by mengstr** [reply]
Edit | 'Circuit specific settings' and then the 'Advanced' tab

<img width="587" alt="Screenshot 2022-07-12 at 21 25 34" src="https://user-images.githubusercontent.com/325326/178577880-6e425d7c-5c5d-4110-b27c-3133195d2c70.png">

**Reply by jongena** [reply]
Many thanks again to all for the perfect answers ! Thank you very much !
